test/unittest/extension/omatcopy_test.cpp

```diff
@@ -0,0 +1,108 @@
+/***************************************************************************
+ *
+ *  @license
+ *  Copyright (C) Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCL-BLAS: BLAS implementation using SYCL
+ *
+ *  @filename omatcopy_test.cpp
+ *
+ **************************************************************************/
+
+#include "blas_test.hpp"
+#include "extension_reference.hpp"
+
+using index_t = int;
+
```

Comment:
```suggestion
```
I think there is no need of this.

*The same applies for the other tests.*

Reply:
yes, it also breaks the test if `index_t` is not `int`. I will update it